scripts/docs/haddocks.sh

```diff
@@ -1,38 +1,35 @@
 #!/bin/bash
```

Comment:
As `/bin/bash` does not exist on NixOS:
```suggestion
#!/usr/bin/env bash
```
so I no longer have to run `bash scripts/docs/haddocks.sh` :laughing:

Reply:
Oh, the PR is already in the merge queue; this change isn't too important, so I won't stop it.